### PR TITLE
remote data uploading functions to Modal Volume 

### PR DESCRIPTION
**Pull request by lordcod99**
added util functions to copy remote data and updated the modal volume data setup file to down load from s3 bucket, @surendhar-palanisamy @pramttl  please review

### CODE REVIEW

File: src/modalvolumeData_setup.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+import logging
+import yaml
+import zipfile
+from dotenv import load_dotenv
+import boto3
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+
+image = modal.Image.debian_slim().run_commands(
+    "pip install gdown",
```
Comment (pramttl) [inline src/modalvolumeData_setup.py:14]: Let's not do run_commands here and use `pip_install`, preferably add these to a requirements.txt file or requirements-modal.txt and reference that file. Modal has a way to install from requirements.txt (see docs).

File: src/modalvolumeData_setup.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
+import modal
```
Comment (pramttl) [inline src/modalvolumeData_setup.py:1]: nit: For the filename, lets keep it in accordance with pip standards, all small case.

File: src/modal_functions/volumeData_setup.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+        # AWS credentials setup
+        self.s3_client = boto3.client(
+            's3',
+            aws_access_key_id=os.getenv('AWS_ACCESS_KEY_ID'),
+            aws_secret_access_key=os.getenv('AWS_SECRET_ACCESS_KEY')
+        )
+
+        self.bucket_name = s3_bucket_name
+        self.files = {
```
Comment (pramttl) [inline src/modal_functions/volumeData_setup.py:26]: nit: `self.files` is a bit ambiguous, we need to use a more explicit var name which tells us what these files are for, for example `files_config_and_maps`.

File: .gitignore [hunk 4]
```diff
@@ -139,3 +140,6 @@ install_manifest.txt
 *.cmake
 *.cmake.in
 *.db
+
```
Comment (pramttl) [inline .gitignore:143]: optional/suggestion: Let's make a `temp` dir or something and add that to .gigitnore and any temporary files used for local testing only can be added to temp.

File: src/modalvolumeData_setup.py [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+s3_client = boto3.client(
+    's3',
+    aws_access_key_id=os.getenv('AWS_ACCESS_KEY_ID'),
+    aws_secret_access_key=os.getenv('AWS_SECRET_ACCESS_KEY')
+)
+
+bucket_name = 'vis4ion'
+
+files = {
+    "demo_query.png": "17MzPE9TyKiNsi6G59rqLCMMd40cIK3bU",
```
[the remark this answers is not in the document]
Comment (lordcod99) [reply src/modalvolumeData_setup.py:35]: there are id's or files, that we download from google drivee

**Comment by lordcod99** [comment]
@surendhar-palanisamy @pramttl , please reiview new changes

**Pull request merged**